content/articles/2019-04-dbplyr-1-4-0.Rmarkdown

```diff
@@ -0,0 +1,185 @@
+---
+title: dbplyr 1.4.0
+author: Hadley Wickham
+date: '2019-04-30'
+slug: dbplyr-1-4-0
+categories:
+  - package
+tags:
+  - databases
+photo:
+  url: https://unsplash.com/photos/JuFcQxgCXwA
+  author: Samuel Zeller
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+```
+
+We're stoked to announce the release of dbplyr 1.4.0, the database backend for dplyr that translates R code into SQL. dbplyr allows you to use a remote database as if it was a collection of local data frames: you write ordinary dplyr code and dbplyr turns it into SQL for you.
```

Comment:
I think we usually link to the pkgdown site at the first mention of the package.